app/models/mms_client.rb

```diff
@@ -40,8 +40,18 @@ def captures_for_item(uuid)
 
   private
 
+  # This DRYs up the pattern of making a request and throwing an exception for bad trsponses
+  def make_request_for(export_type, uuid, params = {})
+    response = authed_request.get(export_url_for(export_type, uuid), params: params)
+    if response.code >= 400
+      throw RuntimeError.new("Error getting #{export_type} for UUID #{uuid}: #{response.code} #{response}")
+    else
+      response.to_s
+    end
+  end
+
   # Builds a URL like http://metadata.nypl.org/exports/mods/123-456
-  def mms_export_of(export_type, uuid)
+  def export_url_for(export_type, uuid)
```

Comment:
I changed this method name to reflect the fact that it's just building a URL string.